sorc/ncep_post.fd/INITPOST_NETCDF.f

```diff
@@ -3135,15 +3125,77 @@ SUBROUTINE INITPOST_NETCDF(ncid2d,ncid3d)
           if (sm(i,j) /= 0.0) runoff(i,j) = spval
         enddo
       enddo
+
+! total water storage in aquifer
+      VarName='wa_acc'
+      call read_netcdf_2d_para(ncid2d,im,jsta,jsta_2l,jend,jend_2u, &
+      spval,VarName,twa)
+!     mask water areas
+!$omp parallel do private(i,j)
+      do j=jsta,jend
+        do i=1,im
+          if (sm(i,j) /= 0.0) twa(i,j) = spval
+        enddo
+      enddo
```

Comment:
you have duplicate section for wa_acc

Reply:
@HuiyaChuang-NOAA Thanks for catching. Will remove it.